docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
+########################################
+
+With the Courses API **Courses** resource, you can complete the
+following tasks.
+
+   * - :ref:`Get a list of all courses in a catalog <Get a List of All Courses
+     in a Catalog>`
+     - GET
+     - /api/v1/catalogs/1/courses/
```

Comment:
Should this just be `/api/v1/courses`? I moved this text out from under the Course Catalog API information.